mui/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v4.0.0 (November 30, 2022)
+
+### Changed
+
+-   Removed dependency on @mui/styles. [#232](https://github.com/brightlayer-ui/icons/issues/232)
+-   Changed `EvPlugIec62196T3a` icon name to `EvPlugIec62196T3A`
+-   Changed `EvPlugIec62196T3c` icon name to `EvPlugIec62196T3C`
```

Comment:
I have no idea what the meta.json does, but the icon is exported as stated

Reply:
> exported as stated

are you exporting the first icon as "EVPlugIEC62196T3A", "as stated" by me above, or are you exporting them as "EvPlugIec62196T3A", as stated by the change log?

Reply:
Sorry for not being clear there, the latter though a.k.a. as it is in the change log. The formatting of the name is all done by the svgr command line tool so I'm just using the default names which match our previous names with the exception of these 2.

Reply:
I was suggesting that:
(1) Leaving it as it is, `EvPlugIec62196T3a`. This is also consistent with our instructions on docit
(2) convert all acronyms, such as "EV", "IEC" and "T3A" into ALL CAPS, so that the icon is named `EVPlugIEC62196T3A`. If you choose this one, then docit's icon instructions would also need to be updated
(3) This proposed `EvPlugIec62196T3A`, as stated in the changelog, has a mix of different naming rules — a mix of ALL CAPS and TitleCase for acronyms. In this case, docit will also need to be updated.

Reply:
1) I'll make the necessary updates to doc-it instructions if need be.
2/3) There's no mix in case from a naming convention, its all PascalCase based on the svg file name. If there are specific needs based on acronyms internally then we should probably discuss that. There isn't a practical way to separate nouns from acronyms with the naming convention unless we do some very specific find and replace. There's also only 3 naming formats that we can use from svgr so any other solution would require an additional script.

Reply:
I am still perplexed. The SVG name `ev_plug_iec_62196_t3a.svg` would be written as`EvPlugIec62196T3a` in PascalCase?

Reply:
I'm assuming it just sees the "a" after the 3 as a new word.